db/migrate/20230505202121_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] I would have liked to approve at the first 👏🏽  👏🏽 👏🏽 if only you have done the same for the like table as for the others. Please use 👉🏽 `t.references :author, null: false, foreign_key: { to_table: 'users' }` instead of  `t.references :user, null: false, foreign_key: true`

Reply:
Thank you for your feedback!